### PR TITLE
Add Healthcare service

### PR DESCRIPTION
**Pull request by pramodya1994**
Resolves; https://github.com/wso2/ballerina-integrator/issues/4

### CODE REVIEW

File: examples/guides/services/hospital-service/daos/healthcare_dao.bal [hunk 1]
```diff
@@ -0,0 +1,55 @@
+type DoctorNotFoundError error<string>;
```
Comment (ThishaniLucas) [inline examples/guides/services/hospital-service/daos/healthcare_dao.bal]: Let's add license header for all the files.

File: examples/guides/services/hospital-service/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# ballerina-hospital-service
```
Comment (ThishaniLucas) [inline examples/guides/services/hospital-service/README.md]: Let's give a small intro about the purpose of this service, what this service does and how to run this service.

File: examples/guides/services/hospital-service/daos/healthcare_dao.bal (hunk 1 above)
Comment (pramodya1994) [reply examples/guides/services/hospital-service/daos/healthcare_dao.bal]: Added

File: examples/guides/services/healthcare-service/.gitignore [hunk 3]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (Asitha) [inline examples/guides/services/healthcare-service/.gitignore]: Shall we  merge this with main .gitginore?

File: examples/guides/services/healthcare-service/Ballerina.toml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+[project]
+org-name = "pramodya"
```
Comment (Asitha) [inline examples/guides/services/healthcare-service/Ballerina.toml]:
```suggestion
org-name = "wso2"
```

File: examples/guides/services/healthcare-service/.gitignore (hunk 3 above)
Comment (pramodya1994) [reply examples/guides/services/healthcare-service/.gitignore]: Done

**Pull request merged**